File: BluRingProFront/src/models/AsistenciaResumen.ts
export interface AsistenciaResumen {
  idResumen: number;
  idEmpleado: number;
  nombreEmpleado: string;
  departamento: string; // ✅ agregado
  fecha: string;        // DateOnly -> string
  horaEntrada?: string; // TimeOnly -> string "HH:mm:ss"
  horaSalida?: string;
  horasLaboradas?: number;
  horasPermiso?: number;
  createdAt: string;
}

File: Blu Ring Pro/Models/AsistenciaResumen.cs
using System;
using System.Collections.Generic;

namespace Blu_Ring_Pro.Models;

public partial class AsistenciaResumen
{
    public int IdResumen { get; set; }

    public int IdEmpleado { get; set; }

    public string NombreEmpleado { get; set; } = null!;

    public DateOnly Fecha { get; set; }

    public TimeOnly? HoraEntrada { get; set; }

    public TimeOnly? HoraSalida { get; set; }

    public decimal? HorasLaboradas { get; set; }

    public decimal? HorasPermiso { get; set; }

    public DateTime CreatedAt { get; set; }

    public virtual Empleados IdEmpleadoNavigation { get; set; } = null!;
}
